### ISSUE TITLE
Custom Converter is ignored when serializing objects in list

### ISSUE THREAD

**Issue by oharaandrew314** [cbeust/klaxon#84]
Custom Converters not used for items in lists.  This does not seem to apply to nested objects, which work as intended.

```
/**
     * This test passes.
     */
    @Test
    fun serializeNestedInstant() {
        data class Person(val firstName: String, val dob: Instant)

        class EpochMilliInstantConverter: Converter<Instant> {
            override fun toJson(value: Instant) = value.toEpochMilli().toString()
            override fun fromJson(jv: JsonValue) = throw NotImplementedError()
        }

        // When no custom converted is provided, empty value for dob.  This used to throw an exception.  Not sure what intended operation is
        val obj = Person("John", Instant.ofEpochMilli(9001))
        Assert.assertThat(Klaxon().toJsonString(obj), CoreMatchers.not(CoreMatchers.containsString("9001")))
        // Actual: {"dob" : {}, "firstName" : "John"}

        // When custom converted is provided, dob is serialized as expected
        val mapper = Klaxon().converter(EpochMilliInstantConverter())
        Assert.assertThat(mapper.toJsonString(obj), CoreMatchers.containsString("9001"))
        // Actual: {"dob" : 9001, "firstName" : "John"}
    }

    /**
     * This test does not pass.
     */
    @Test
    fun serializeListOfInstants() {
        val dates = listOf(Instant.ofEpochMilli(9001))

        class EpochMilliInstantConverter: Converter<Instant> {
            override fun toJson(value: Instant) = value.toEpochMilli().toString()
            override fun fromJson(jv: JsonValue) = throw NotImplementedError()
        }

        // despite custom converter being provided, instant is not serialized.  Empty value in list
        val mapper = Klaxon().converter(EpochMilliInstantConverter())
        Assert.assertThat(mapper.toJsonString(dates), CoreMatchers.containsString("9001"))
        // Actual: [{}]
    }
```

I appreciate all of your quick resolutions so far! 👍

**Comment by cbeust** [comment]
Fixed in 2.0.10.

**Comment by oharaandrew314** [comment]
Confirmed as fixed.
